pdfminer/pdfdocument.py

```diff
@@ -99,7 +99,7 @@ def load(self, parser):
                 raise PDFNoValidXRef('Unexpected EOF - file corrupted?')
             if not line:
                 raise PDFNoValidXRef('Premature eof: %r' % parser)
-            if line.startswith(b'trailer'):
+            if line.strip().startswith(b'trailer'):
```

Comment:
Everytime `line` is used it is stripped, so I think its best to strip the whitespace right after parsing on line 95.